modules/shared/components/atoms/Box/IBox.d.ts

```diff
@@ -0,0 +1,13 @@
+import type { ReactElement } from 'react';
+
+declare namespace IBox {
+  export interface IProps {
+    classes?: string;
```

Comment:
What is the use for the Classes prop?

Reply:
In case we wanted to apply additional classes.

Reply:
do you test this className prop? do we use it at the moment? if the answer is No, then probably we should remove it and introduce it when we have the first use case